staging/src/k8s.io/apiserver/plugin/pkg/audit/webhook/webhook.go

```diff
@@ -229,15 +291,43 @@ L:
 		}
 	}
 
+	return events
+}
+
+// collectLastEvents assumes that the buffer was closed. It collects the first
+// maxBatchSize events from the closed buffer into a batch and returns them.
+func (b *batchBackend) collectLastEvents() []auditinternal.Event {
```

Comment:
I'm sorry! I surely understand you argument and totally agree with that. What I wanted to say is that it doesn't make much sense to extract this variable, since this method is going to be used in one place and the method extracting was done more to emphasize the purpose of this piece of code, compared to regular `collectEvents`. Do you agree or do you think it will help to the code readability in this particular place to make this method a separate function?